<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(85, 85, 128);
</string>
  </property>
  <widget class="QComboBox" name="category_input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <item>
    <property name="text">
     <string>Phone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Laptop</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>731</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_bt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Delivery Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="brand_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>361</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <item>
    <property name="text">
     <string>Acer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GigaByte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Asus</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Msi</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="color_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>421</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pink</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="date_input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_bt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantity_input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>421</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_bt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_name_input">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_bt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_bt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_bt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 170, 127);
border-radius:10px;
color:white;</string>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>470</y>
     <width>731</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <widget class="QLabel" name="validator_label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>15</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
